### ISSUE TITLE
Paginator: don't write element if not template

### ISSUE THREAD

**Issue by Rapster** [primefaces/primefaces#2708]
So far, there is predefined template aka "paginator elements", all surrounded by `{}`. If a template is defined in paginatorTemplate but not defined, his name is written in output stream. IMO, if defined as a template but not found, nothing should be written in output. You might ask about text you'd like to write after a template, in that case, define as a static text, means no `{}`.

My use case is simple, I have a datatale in a composite defining custom facets, those are not defined sometimes depending the page but I definitley don't want see the name of the facet written in the paginator.

To sum up:

- `{}` define a template/facet, if not found not written
- if no `{}` set, it's written as plain text

**Comment by tandraschko** [comment]
+1

**Comment by Rapster** [comment]
Great, I'll provide a PR then

**Comment by tandraschko** [comment]
Does it break something?

**Comment by Rapster** [comment]
Nop, the condition covers only this point:

- `{}` define a template/facet, if not found not written

**Comment by kukel** [comment]
Maybe in 'jsf development' mode write a log line that a facet was used but not defined/declared

**Comment by tandraschko** [comment]
@kukel @Rapster a log would be a good small enhancement. If anyone has time, please prepare a PR.
